DfE.FindInformationAcademiesTrusts/Pages/Trusts/Overview.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@page
+@model OverviewModel
+
+@{
+  Layout = "_TrustLayout";
+}
+
+<section>
+  <div class="govuk-!-margin-top-9 govuk-!-margin-bottom-1">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" aria-labelledby="trust-information">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="trust-information">Trust summary</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <dl class="govuk-summary-list">
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Total academies
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Academies in each local authority
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  <p class="govuk-!-margin-bottom-1">3 in North Lincolnshire</p>
+                  <p class="govuk-!-margin-bottom-1">1 in North East Lincolnshire</p>
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil numbers
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4,896
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil capacity <br>(% full)
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  6,463<br>(76%)
+                </dd>
+              </div>
+            </dl>
+          </div>
+        </section>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" aria-labelledby="ofsted-ratings">
```

Comment:
Same here, this can be removed and you can use aria-describedby for the table :)